Documentation/Stage-1-Introduction/Day-1/Index.rst

```diff
@@ -42,9 +42,10 @@ Theoretical prerequisites
 
 This day assumes that you already know the following:
 
-* Item 1
-* Item 2
-* Item 3
+* Basic knowledge of web technologies
+* Basic understanding of CMS concepts
+* Basic understanding of what is TYPO3
+* Basic understanding of what is the Bootstrap framework
```

Comment:
The three final points sound like goals for this day, so I would move them to the theoretical goals section.